mrio-log/src/MRIO/Log.hs

```diff
@@ -34,13 +34,9 @@ import qualified Data.Text.Encoding       as Text
 import qualified Data.Time.Clock          as Clock
 import qualified System.IO                as IO
 
-type Action m = Message -> m ()
+newtype Action = Action (Message -> IO ())
```

Comment:
Nitpick (not-blocking): 

Making Action a new-type has a batch of advantages that I can think of through adding type class instance, however, we are not using that power yet.
 
So this makes me wonder whether leaving it as a type would have been a viable alternative or there is a stronger reason for changing to new-type that I may be missing.

Reply:
I agree it should have been just a type alias, artifact from me refactoring from an intermediate type where I hid a `forall`.